docs/howto/account-billing/change-billing-data.md

```diff
@@ -0,0 +1,30 @@
+# Changing your account billing data
+
+Via [{{gui}}](https://{{gui_domain}}) you can change the contact
+person, address, company name, and purchase order number associated
+with your account.
+
+To get started, navigate to <https://{{gui_domain}}>.  Log in and
```

Comment:
Just an extra space in "[...]<https://{{gui_domain}}>.  Log in and[...]" :)

Reply:
Fixed, thanks.